Pokedex/MainScreen.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ViewController.swift
+//  Pokedex
+//
+//  Created by Стефан  Миливоевич on 5.10.22..
+//
+
+import UIKit
+
+class MainScreen: UIViewController, UITableViewDelegate, UITableViewDataSource  {
```

Comment:
Давай контроллер оставим чистеньким, только основную логику по отображению view
```suggestion
class MainScreen: UIViewController  {
```
А функционал, связанный с расширением через протоколы, можно вынести в extension ниже, так как файл пока небольшой
```suggestion
extension MainScreen: UITableViewDelegate, UITableViewDataSource  {
```